includes/Integrations/Site_Kit.php

```diff
@@ -0,0 +1,167 @@
+		add_filter( 'googlesitekit_amp_gtag_opt', [ $this, 'filter_site_kit_gtag_opt' ] );
+
+		if ( in_array( 'analytics', $this->get_site_kit_active_modules_option(), true ) ) {
+			remove_action( 'web_stories_print_analytics', [ $this->analytics, 'print_analytics_tag' ] );
+		}
+	}
+
+	/**
+	 * Filters Site Kit's Google Analytics configuration.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param array $gtag_opt Array of gtag configuration options.
+	 *
+	 * @return array Modified configuration options.
+	 */
+	public function filter_site_kit_gtag_opt( $gtag_opt ) {
+		if ( ! is_singular( Story_Post_Type::POST_TYPE_SLUG ) ) {
+			return $gtag_opt;
+		}
+
+		$default_config             = $this->analytics->get_default_configuration( $gtag_opt['vars']['gtag_id'] );
+		$default_config['triggers'] = isset( $default_config['triggers'] ) ? $default_config['triggers'] : [];
+
+		$gtag_opt['triggers'] = isset( $gtag_opt['triggers'] ) ? $gtag_opt['triggers'] : [];
+		$gtag_opt['triggers'] = array_merge(
+			$default_config['triggers'],
+			$gtag_opt['triggers']
+		);
+
+		return $gtag_opt;
+	}
+
+	/**
+	 * Gets the option containing the active Site Kit modules.
+	 *
+	 * Checks two options as it was renamed at some point in Site Kit.
+	 *
+	 * Bails early if the Site Kit plugin itself is not active.
+	 *
+	 * @see \Google\Site_Kit\Core\Modules\Modules::get_active_modules_option
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array List of active module slugs.
+	 */
+	private function get_site_kit_active_modules_option() {
+		if ( ! defined( 'GOOGLESITEKIT_VERSION' ) ) {
+			return [];
+		}
+
+		$option = get_option( 'googlesitekit_active_modules' );
+
+		if ( is_array( $option ) ) {
+			return $option;
+		}
+
+		$legacy_option = get_option( 'googlesitekit-active-modules' );
+
+		if ( is_array( $legacy_option ) ) {
+			return $legacy_option;
+		}
+
+		return [];
+	}
+
+	/**
+	 * Returns the Site Kit plugin status.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array Plugin status.
+	 */
+	public function get_plugin_status() {
+		$is_installed        = array_key_exists( 'google-site-kit/google-site-kit.php', get_plugins() );
+		$is_active           = defined( 'GOOGLESITEKIT_VERSION' );
+		$is_analytics_active = in_array( 'analytics', $this->get_site_kit_active_modules_option(), true );
+
+		$link      = __( 'https://wordpress.org/plugins/google-site-kit/', 'web-stories' );
+		$dashboard = admin_url( 'admin.php?page=googlesitekit-dashboard' );
+		$settings  = admin_url( 'admin.php?page=googlesitekit-settings' );
+
+		if ( $is_analytics_active ) {
+			if ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
+				$link = $dashboard;
+			}
+		} elseif ( $is_active ) {
+			if ( current_user_can( 'googlesitekit_manage_options' ) ) {
+				$link = $settings;
+			} elseif ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
+				$link = $dashboard;
+			}
+		} elseif ( $is_installed ) {
+			if ( current_user_can( 'activate_plugin', 'google-site-kit/google-site-kit.php' ) ) {
```

Comment:
Do we need to have a check if you can enable network active plugin as well?

Reply:
I think `map_meta_cap` does that? Checks for `manage_network_plugins`